[mesa] Unable to build

```
./rasterizer/jitter/gen_builder.hpp:76:97: error: invalid conversion from ‘llvm::Value*’ to ‘unsigned int’ [-fpermissive]
     return IRB()->CreateMemMove(Dst, Src, Size, Align, isVolatile, TBAATag, ScopeTag, NoAliasTag);
                                                                                                 ^
In file included from ./rasterizer/jitter/jit_pch.hpp:46:0,
                 from ./rasterizer/jitter/JitManager.h:32,
                 from swr_shader.cpp:27:
/usr/local/include/llvm/IR/IRBuilder.h:494:13: note:   initializing argument 2 of ‘llvm::CallInst* llvm::IRBuilderBase::CreateMemMove(llvm::Value*, unsigned int, llvm::Value*, unsigned int, uint64_t, bool, llvm::MDNode*, llvm::MDNode*, llvm::MDNode*)’
   CallInst *CreateMemMove(Value *Dst, unsigned DstAlign, Value *Src, unsigned SrcAlign,
             ^~~~~~~~~~~~~
In file included from ./rasterizer/jitter/builder.h:157:0,
                 from swr_shader.cpp:35:
./rasterizer/jitter/gen_builder.hpp: In member function ‘llvm::Value* SwrJit::Builder::INVARIANT_GROUP_BARRIER(llvm::Value*)’:
./rasterizer/jitter/gen_builder.hpp:981:19: error: ‘class llvm::IRBuilder<>’ has no member named ‘CreateInvariantGroupBarrier’; did you mean ‘CreateInvariantStart’?
     return IRB()->CreateInvariantGroupBarrier(Ptr);
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~
                   CreateInvariantStart
  CXX      libmesaswr_la-swr_loader.lo
  CXX      rasterizer/common/libmesaswr_la-formats.lo
make[5]: *** [Makefile:2883: libmesaswr_la-swr_shader.lo] Error 1
make[5]: *** Waiting for unfinished jobs....
make[5]: Leaving directory '/usr/local/tmp/crew/mesa-18.3.1.tar.xz.dir/mesa-18.3.1/src/gallium/drivers/swr'
make[4]: *** [Makefile:1891: all] Error 2
make[4]: Leaving directory '/usr/local/tmp/crew/mesa-18.3.1.tar.xz.dir/mesa-18.3.1/src/gallium/drivers/swr'
make[3]: *** [Makefile:634: all-recursive] Error 1
make[3]: Leaving directory '/usr/local/tmp/crew/mesa-18.3.1.tar.xz.dir/mesa-18.3.1/src/gallium'
make[2]: *** [Makefile:895: all-recursive] Error 1
make[2]: Leaving directory '/usr/local/tmp/crew/mesa-18.3.1.tar.xz.dir/mesa-18.3.1/src'
make[1]: *** [Makefile:680: all] Error 2
make[1]: Leaving directory '/usr/local/tmp/crew/mesa-18.3.1.tar.xz.dir/mesa-18.3.1/src'
make: *** [Makefile:692: all-recursive] Error 1
mesa failed to install: `make V=0 -j2` exited with 2
```

--- a/packages/mesa.rb
+++ b/packages/mesa.rb
@@ -13,22 +13,14 @@ class Mesa < Package
   })
 
   depends_on 'libdrm'
-  depends_on 'xorg_proto'
-  depends_on 'libx11'
-  depends_on 'libxext'
-  depends_on 'libxfixes'
-  depends_on 'libxdamage'
-  depends_on 'libxcb'
-  depends_on 'libxvmc'
-  depends_on 'libxtrans'
-  depends_on 'libxkbfile'
   depends_on 'wayland'
-  depends_on 'python27'
-  depends_on 'bison'
-  depends_on 'libxshmfence'
   depends_on 'elfutils'
   depends_on 'llvm'
-  
+
+  def self.patch
+    system "wget -O- https://patchwork.freedesktop.org/patch/257229/raw | patch -Rp1"
+  end
+
   def self.build
     system "pip uninstall -y Mako MarkupSafe || :"
     system "pip install --prefix \"#{CREW_PREFIX}\" --root \"#{CREW_DEST_DIR}\" Mako==1.0.7"

